packages/content/src/Infrastructure/Symfony/HttpKernel/SuluContentBundle.php

```diff
@@ -38,7 +38,16 @@ final class SuluContentBundle extends AbstractBundle
      */
     public function configure(DefinitionConfigurator $definition): void
     {
-        $definition->rootNode();
+        $definition->rootNode() // @phpstan-ignore-line
+            ->children()
+                ->arrayNode('resource_resolver')
```

Comment:
@Prokyonn we should maybe reflect the service name here.

```suggestion
                ->arrayNode('content_resolver')
```

Reply:
Sounds good for me 👍

Reply:
Fixed